packages/@aws-cdk/aws-lambda/package.json

```diff
@@ -64,6 +64,7 @@
     "@aws-cdk/aws-logs": "^0.8.2",
```

Comment:
What happened here? Seems like JSON is not formatted well (at least that's what GH thinks)

Reply:
not sure, my local diff doesn't show any abnormality. seems like after a pull and rebase, the existing file is showing up in red too.

Reply:
current HEAD of repository shows red lines all over https://github.com/awslabs/aws-cdk/blob/master/packages/%40aws-cdk/aws-lambda/package.json 

looks like it got introduced here: https://github.com/awslabs/aws-cdk/commit/934ca5f36dce9c4b76513f2c32e8a940bdbd74c1#diff-8efe73f2524a756df86464dcac4836f6